### PR TITLE
Add additional sizes to radial progress indicator

### LINKED ISSUE #79: ProgressRadial: Need additional sizes
Feature:
The iModel.js UI team is now consuming iTwinUI and is deprecating its own Spinner & ProgressSpinner components in favor of the ProgressRadial component in iTwinUI-react. The iModel.js spinner components support 16px, 32px and 64px sizes. The ProgressRadial supports two sizes: small (24px) and unspecified (32px). We are asking for support of all sizes (16px, 24px, 32px and 64px) in the ProgressRadial component.

Examples:
&lt;ProgressRadial size="tiny" /&gt;
&lt;ProgressRadial size="small" /&gt;
&lt;ProgressRadial size="medium" /&gt; is same as &lt;ProgressRadial /&gt;
&lt;ProgressRadial size="large" /&gt;

### PR DESCRIPTION
**Pull request by FlyersPh9**
- Added `.iui-x-small` and `.iui-large` sizes to radial progress indicator.  Closes #79 .
- `.iui-x-small` does not support inner content.
- Refined tests to exclude parts that were not used / a part of what we wanted to test.
Screen Shot 2021-05-19 at 5 58 44 PM

### CODE REVIEW

File: backstop/scenarios/progress-indicator.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ module.exports = [
   // Progress indicator types
   scenario('linear', {
     selectors: ['#demo-linear'],
-    hideSelectors: ['.iui-indeterminate'],
+    hideSelectors: ['.iui-indeterminate', 'h3'],
```
Comment (bentleyvk) [inline backstop/scenarios/progress-indicator.js:7]: Maybe better change placement of `<section>`. I see in other tests we wrap with section only test elements and headers are outside.

File: src/progress-indicator/radial.scss [hunk 2]
```diff
@@ -218,3 +237,8 @@
     }
   }
 }
+
+@mixin iui-progress-indicator-radial-size($size:$iui-icons-xl) {
```
Comment (bentleyvk) [inline src/progress-indicator/radial.scss:241]: Is it worth having default value? This mixin is only used in this file so I would say explicitly pass size in default case.

File: backstop/scenarios/progress-indicator.js (hunk 1 above)
Comment (FlyersPh9) [reply backstop/scenarios/progress-indicator.js:7]: The red boxed in area is the `<section>`.  I did place the `<h2>` outside, consistent with other tests, but I left the `<h3>` inside and hid them.  The alternative would be to have each section under an `<h3>` within their own `<section>` but that would introduce more test images.
Screen Shot 2021-05-20 at 11 01 44 AM

File: src/progress-indicator/radial.scss [hunk 3]
```diff
@@ -169,13 +167,34 @@
     }
   }
 
+  &.iui-x-small {
```
[the remark this answers is not in the document]
Comment (FlyersPh9) [reply src/progress-indicator/radial.scss:170]: We're using `x-large` for user icons so I thought `x-small` would stay consistent.

File: src/progress-indicator/radial.scss (hunk 2 above)
Comment (FlyersPh9) [reply src/progress-indicator/radial.scss:241]: Makes sense, I have removed the default value.

**Review by bentleyvk — APPROVED** [review]

**Review by veekeys — APPROVED** [review]

**Review by mayank99 — APPROVED** [review]

**Pull request merged**